sdk/textanalytics/tests.yml

```diff
@@ -1,5 +1,12 @@
 trigger: none
 
+resources:
```

Comment:
why is this needed?

Reply:
It should not added. I was looking at the .NET's older commint (through the Blame https://github.com/Azure/azure-sdk-for-net/blame/3c6a978b247cce2f071dfa57db2288fb0882e8ca/sdk/textanalytics/tests.yml),